tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+-  **Hardware Tenancy:** this option allows selection of a dedicated instance type for the VPN gateway.  Use the default option.
+
+After completing the form, click **Next** to proceed to Step 3.
+
+### Configuring the VPN
+
+To configure the VPN, enter the **Customer Gateway IP, **which is the IP address assigned to the Google Cloud Platform VPN gateway created in the [Configuration - GCP section](https://docs.google.com/document/d/1lKwKy-Zx66ztS5s11llA1w2TmAteLCpqng3Lh5VzfCw/edit?ts=5ac3b3a7#heading=h.d6lxerw0qhtv):
+
+![image](cloudvpnguide---g6bgr4s6x3.png)
+
+In addition to the Customer Gateway IP, enter a **Customer Gateway name** and a **VPN Connection name**.
+
+Next, choose a Routing Type for the VPN connection.  This section of the guide covers VPN with BGP route management, so select **Dynamic**.  Enter the Google Cloud Platform subnet CIDR block under **IP Prefix** and click **Add**: 
+
+![image](cloudvpnguide---w8aoc7kglhs.png)
+
+When all required configuration is completed, click **Create VPC** to create the new VPC network and finish the Wizard.  VPC network creation takes a minute or two to complete.  Once completed, the management console status is updated:
+
+![image](cloudvpnguide---9adcawwbfs.png)
+
+Select the newly-created VPC  network from the Dashboard to collect the configuration detail required to complete the [GCP configuration](https://docs.google.com/document/d/1lKwKy-Zx66ztS5s11llA1w2TmAteLCpqng3Lh5VzfCw/edit?ts=5ac3b3a7#heading=h.d6lxerw0qhtv):
+
+![image](cloudvpnguide---6cmfivaepqe.png)
+
+AWS requires two tunnels for redundancy.  The last step is to collect the IP addresses of the AWS Virtual Gateway and the pre-shared keys used for IKE authentication that are automatically generated by AWS.  These configuration details can be downloaded by clicking **Download Configuration**.  Several device-specific options are available for the configuration format.  For GCP, select **Generic**:
+
+![image](cloudvpnguide---ifcrcccs1zj.png)
+
+![image](cloudvpnguide---ck2ns7co8fm.png)
+
+The configuration file is an ASCII text file.  Note that the auto-generated pre-shared key is listed under **Pre-Shared Key **and can't be user defined.  The link local address for BGP peering islisted under **Inside Addresses** and also can't be user defined.  
+
+## Configuration - GCP
+
+Google Cloud Router enables dynamic [Border Gateway Protocol (BGP)](https://www.wikipedia.org/wiki/Border_Gateway_Protocol) route updates between your Google Cloud Platform network and your on-premise network. Cloud Router works with both legacy networks and [subnets](https://cloud.google.com/compute/docs/subnetworks).  
+
+### Configuring the VPN Tunnel
+
+Use  t[he VPN creation page](https://console.cloud.google.com/vpn) to create the Cloud VPN gateway and tunnels. AWS requires two tunnels for redundancy.
+
+Use the following parameters to configure the Cloud VPN gateway:
+
+-  **Name: **the name of the VPN gateway.
```

Comment:
Remove space between "Name" and "**"
In all of these, you have a space that breaks the formatting.